ci/ray_ci/doc/BUILD.bazel

```diff
@@ -14,6 +14,13 @@ py_binary(
     exec_compatible_with = ["//:hermetic_python"],
 )
 
+py_binary(
+    name = "load_doc_cache",
```

Comment:
similary, use `cmd_` prefix for the binary

Reply:
done!